python/ray/autoscaler/_private/command_runner.py

```diff
@@ -765,14 +748,45 @@ def run_init(self, *, as_head, file_mounts):
                 for remote, local in cleaned_bind_mounts.items():
                     remote = self._docker_expand_user(remote)
                     if remote not in active_remote_mounts:
-                        cli_logger.error(
-                            "Please ray stop & restart cluster to "
-                            f"allow mount {remote}:{local} to take hold")
+                        try:
+                            is_container_running = not cli_logger.confirm(
+                                False,
+                                "This Docker Container is already Running, Do "
+                                "you want to restart the Docker container on "
+                                "this node?")
```

Comment:
this will always fail i think on the worker nodes since it requires interactivity and that's not available when going from head -> worker. Can you default to 'True'?

Reply:
I will put that in the `except block` because the "True" here is for whether to skip the 'confirm'.